src/ui/public/timepicker/timepicker.js

```diff
@@ -24,7 +24,9 @@ module.directive('kbnTimepicker', function (quickRanges, timeUnits, refreshInter
       to: '=',
       mode: '=',
       interval: '=',
-      activeTab: '='
+      activeTab: '=',
+      onFilterSelect: '=',
```

Comment:
It would not be possible to pass arguments like `from` and `to` if we used `onFilterSelect: '&'`. This looks good to React-trained eyes. :wink: